includes/class-wc-stripe-api.php

```
@@ -533,16 +537,10 @@ public static function update_payment_method( $payment_method_id, $payment_metho
 	public static function attach_payment_method_to_customer( string $customer_id, string $payment_method_id ) {
 		// Sources and Payment Methods need different API calls.
 		if ( 0 === strpos( $payment_method_id, 'src_' ) ) {
```

PHPMD (2.15.0)

[error] The variable $payment_method_id is not named in camelCase. (undefined)

(CamelCaseVariableName)